docs/troubleshooting/clean-install.md

```diff
@@ -49,15 +49,15 @@ Next you'll need to set up your environment.
    streamlit run myfile.py
    ```
 
-## Install Streamlit on macOS/Linux
+## Install Streamlit on MacOS/Linux
```

Comment:
@tvst + @kellyamanda - Apple switched this again to macOS. We might want to switch this back to how it was: https://www.apple.com/macos/catalina/